### PR TITLE
Feature/en/npm release

### PR DESCRIPTION
**Pull request by eNepper**
Added GitHub action that will run when a new release is created here on github. The version number is taken from the tag created.

### CODE REVIEW

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# This workflow will publish the release as an NPM package
+
+name: Node.js Package
```
Comment (ammapspeople) [inline .github/workflows/npm-publish.yml:3]: Make name great.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # eslint-config-mapsindoors
 
-ESLint [shareable config](http://eslint.org/docs/developer-guide/shareable-configs.html) used in MapsIndoors JavaScript projects.
+ESLint [configuration](http://eslint.org/docs/developer-guide/shareable-configs.html) used within MapsPeople A/S
```
Comment (ammapspeople) [inline README.md:3]: Should this be reverted back into something about MapsIndoors?

File: .github/workflows/npm-publish.yml (hunk 1 above)
Comment (eNepper) [inline .github/workflows/npm-publish.yml:3]:
```suggestion
name: Publish NPM package
```

Comment (ammapspeople) [reply .github/workflows/npm-publish.yml:3]: 👍

File: README.md (hunk 2 above)
Comment (eNepper) [reply README.md:3]: Lets do that the day we become MapsIndoors

Comment (ammapspeople) [reply README.md:3]: Ok.

**Review by ammapspeople — APPROVED** [review]

**Pull request merged**